polyfill/README.md

```diff
@@ -0,0 +1,41 @@
+# Temporal Polyfill
+
+This is an implementation of a polyfill for the [TC39 Termporal Proposal](https://github.com/tc39/proposal-temporal).
+
+## Implementation Issues:
+
+### Edge-Case: Getting *epoch-milliseconds*
+
+Getting the correctly time-zoned *milliseconds since epoch* for a `CivilDateTime` when calling `withZone` is currently
+based on `Intl.DateTimeFormat` fumbling. However this fails for the edgecase where the instant to be extracted falls
+within one timezone offset of a DST-Change.
+
```

Comment:
@icambron do you have a work around for this?

Reply:
Kind of. I don't quite understand why "within one timezone offset of a DST change" is the bounds of the edge case. Maybe the way Luxon uses Intl to do this pretty different?

For Luxon, there are two edge cases:
 
1. The CivilDateTime is inside spring forward and thus doesn't exist. This is easy to detect: guess the offset (Luxon uses "what's the offset for `now()` in that zone?"), compute the epoch millis, then test whether the resulting date has the same offset as your guess. If yes, it's good to go, if not, adjust the epoch millis by the difference. The way Luxon does this, you end up pushing the date forward in these cases. The code for that is (mostly) here:  https://github.com/moment/luxon/blob/baf63e639b1b635f10ef54744c2def23ecfb9140/src/datetime.js#L72.

2. The CivilDateTime is an ambiguous fall back time. This one is tougher. Here's what Luxon does: https://moment.github.io/luxon/docs/manual/zones.html#ambiguous-times. Codewise, that just uses the code above; if the offset of the generated time matches the guess, we're done. That means we could end up with either side of the ambiguity (just by having different guesses) and never even know it, but c'est la vie.